### ISSUE TITLE
"Choose your county" search form field doesn't clear when closed and reopened [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#442]
When an entry is made into the "Chose your county" search field, with autocomplete, if the user decides to close the overlay, the next time the user opens that overlay, the input field retains the user's previous entry, and the search field's label and hint aren't announced, but instead the previous search text (for example, "Xx") by TalkBack

**Comment by jonnyjames** [comment]
This issue persists in latest build (Monday, September 21, 2020)

**Comment by floridemai** [comment]
@jonnyjames Should be fixed in the next release
